serving/docker/partition/partition.py

```diff
@@ -202,7 +208,9 @@ def run_partition(self) -> str:
             self.cleanup()
             return partition_stdout
         else:
-            raise Exception("Partitioning was not successful.")
+            logging.error(
```

Comment:
Log and Return the exit code as well

Reply:
Added
https://github.com/deepjavalibrary/djl-serving/blob/2f131026b30d1c4ea72641b395e2a1b0b5341b14/serving/docker/partition/partition.py#L209-L214

Example output in Neo
```
OptimizationFatalError('Partitioning exited with return code: 1. Details: KeyError: "Unsupported model: 'RefinedWebModel'. Supported architectures: ['opt', 'gpt2', 'gptj', 'gpt_neox', 'llama', 'mistral', 'mixtral', 'bloom']"')
```